tailwind.config.js

```diff
@@ -42,3 +42,9 @@ module.exports = {
         }
     }
 };
+
+if (process.env.NODE_ENV === 'production') {
+    tailwindConfig.content.push('!./src/**/Demo*.vue')
```

Comment:
🔥 

We probably don't need this in staging, either.

Reply:
there's no staging only development (when the demo board is running) and production (when the package is building)